incubator/prometheus/templates/alertmanager-ingress.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.alertmanager.ingress.enabled -}}
+{{- $releaseName := .Release.Name -}}
+{{- $servicePort := .Values.server.httpPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+{{ toYaml .Values.alertmanager.ingress.annotations | indent 4 }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: alertmanager
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "alertmanager.fullname" . }}
+spec:
+  rules:
+  {{- range .Values.alertmanager.ingress.hosts }}
+    - host: {{ .hostName }}
+      http:
+        paths:
+          - path: /
```

Comment:
I'm happy to leave it for now. This is absolutely a non-blocker. Happy to leave it out of this PR unless someone has a compelling reason